bots/activity.bot.js

```diff
@@ -15,14 +15,19 @@ const THRESHOLD_SECONDS = _.parseInt(
 const COOLDOWN_SECONDS = _.parseInt(
 	config.get('activitybot.cooldown_seconds')
 );
+const RUN_INTERVAL_SECONDS = _.parseInt(
+	config.get('activitybot.run_interval_seconds')
+);
 
 const IGNORED_CHANNELS = config.get('activitybot.ignored_channels').split(',');
 
 const coolingChannels = {};
-const messageTimestamps = {};
+let messageTimestamps = {};
 
 
 const unixNow = ()=>Math.floor(Date.now() / 1000);
+const debug(...args)=>DEBUG && Slack.log(args.join(' '))
```

Comment:
Can you actually define & name arrow functions like this? Without actually assigning it with `=`?

Reply:
[nope]([yep](https://github.com/stolksdorf/slack-higgins/commit/9c2e633db013f97bb8d48680cb351e29c5bf606a))